ansible/roles/sonarqube/tasks/check_prerequisites.yml

```diff
@@ -15,13 +15,23 @@
 
 # Check the user specific limits
 - name: Check if user {{ sonar_user }} file descriptors less than 131072
-  command: "su - {{ sonar_user }} --shell /bin/bash -c \"ulimit -n\" 2> /dev/null"
+  become: true
```

Comment:
Do we need to be checking these if we're already configuring it?

Reply:
When developing it, there was several times the users configuration wouldnt always change so having the checks made sure I could ensure it was changing. They arnt required so much now that the Ansible is consitantly applying them but also acts as an extra fail safe anyway.

Reply:
Removed